src/lib/components/schedule/schedule-form-view.svelte

```diff
@@ -94,6 +98,7 @@
       type: $customSearchAttributes[label],
     }),
   ) as SearchAttributeInput[];
+  let workflowSearchAttributesInput: SearchAttributeInput[] = [];
```

Comment:
We'll need to do something like we do for the above indexedFields const so we can get the current workflow search attributes when we edit the schedule instead of always having an empty array

For schedules search attributes we do:
```
  const decodedSearchAttributes = decodePayloadAttributes({ searchAttributes });
  const indexedFields =
    decodedSearchAttributes?.searchAttributes.indexedFields ??
    ({} as { [k: string]: string });
```

For workflow attributes something like:
```
  const decodedWorkflowSearchAttributes = decodePayloadAttributes({ searchAttributes: schedule.action.startWorkflow.searchAttributes });
  const workflowSearchAttributesInput =
    decodedWorkflowSearchAttributes?.searchAttributes.indexedFields ??
    ({} as { [k: string]: string });
```